packages/shared/src/hooks/translation/useTranslation.ts

```diff
@@ -90,23 +147,13 @@ export const useTranslation: UseTranslation = ({ queryKey, queryType }) => {
   const fetchTranslations = useCallback(
     async (posts: Post[]) => {
       if (!isStreamActive) {
-        return;
+        return [];
       }
       if (posts.length === 0) {
-        return;
+        return [];
       }
 
-      const postIds = posts
-        .filter((node) =>
-          node?.title
-            ? !node?.translation?.title
-            : !node?.sharedPost?.translation?.title,
-        )
-        .filter((node) =>
-          flags?.clickbaitShieldEnabled && node?.title
-            ? !node.clickbaitTitleDetected
-            : !node.sharedPost?.clickbaitTitleDetected,
-        )
```

Comment:
We don't need this filtering anymore?

Reply:
I think I translated them into new conditions